src/utils/event.ts

```diff
@@ -42,3 +47,30 @@ export const eventHappeningNow = (
       event.start_utc_seconds + event.duration_minutes > currentTimeInUTCSeconds
   );
 };
+
+export const eventStartTime = (event: VenueEvent) =>
+  fromUnixTime(event.start_utc_seconds);
+
+export const eventEndTime = (event: VenueEvent) =>
+  addMinutes(eventStartTime(event), event.duration_minutes);
+
+export const isEventStartingSoon = (event: VenueEvent) =>
+  differenceInHours(eventStartTime(event), Date.now()) > 0;
```

Comment:
As referred to in other comments in this review + previous review thread, I don't believe the logic in this function is correct for it's rename, and probably needs to be reversed.

Reply:
[note] Old logic of this was in `canUserJoinTheEvent` in `src/utils/time.ts`

Reply:
fixed, see the discussion here: https://github.com/sparkletown/sparkle/pull/1302#discussion_r634215959